src/components/molecules/button-group/button-group.md

```diff
@@ -0,0 +1,45 @@
+```meta
+  category: Layout
+  description: Use a ButtonGroup to put related buttons together
+```
+
+`import ButtonGroup from 'cosmos/button-group'`
+
+---
+
+```jsx
+<ButtonGroup {props}>
+  <Button primary>Save changes</Button>
+  <Button>Clear</Button>
+</ButtonGroup>
+```
+
+## Examples
+
+```js
+<Stack>
+  <ButtonGroup>
+    <Button primary>Save changes</Button>
+    <Button>Clear</Button>
+  </ButtonGroup>
+  <ButtonGroup>
+    <Button icon="pencil" />
+    <Button icon="copy" />
+    <Button icon="trash" />
+  </ButtonGroup>
+</Stack>
+```
+
+```js
+<Stack>
+  <ButtonGroup compressed>
+    <Button primary>Save changes</Button>
```

Comment:
Let's remove the primary button from this example, so they are both default. Usually when compressing the buttons, they have all the same importance.